django_seo_js/backends/base.py

```diff
@@ -46,7 +41,7 @@ class RequestsBasedBackend(object):
 
     def __init__(self, *args, **kwargs):
         super(RequestsBasedBackend, self).__init__(*args, **kwargs)
-        self.requests = requests
+        self.session = requests.Session()
```

Comment:
When using a `Session`, `requests` will take advantage of reusing a connection with keep alives.